app/models/agents/rss_agent.rb

```diff
@@ -113,7 +113,7 @@ def check_urls(urls)
 
       created_event_count = 0
       sort_events(new_events).each.with_index do |event, index|
-        entry_id = event.payload[:id]
+        entry_id = event.payload[:update_id]
```

Comment:
The one thing that will happen is it will re-fetch already seen items when the change is implemented because we're not going to try to migrate the seen_ids to the new format -- but this will just be the one time.

I can't really think of anything else that might be affected...